test/test_translator.py

```diff
@@ -138,8 +209,78 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_null_parameter() :
+
+    test_polyline= None
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    expected_coordinates= None
+    assert  test_coordinates == expected_coordinates
 
-def test_get_road_direction():
+def test_parse_polyline_invalid_data() :
+    test_polyline= 'invalid' 
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    expected_coordinates= []
+    assert  test_coordinates == expected_coordinates
+
+class TestTranslator(unittest.TestCase):
+  def test_parse_polyline_invalid_coordinates(self):
+    test_polyline = 'a,b,c,d'
+    with pytest.raises(RuntimeError) as runtimeErr:
```

Comment:
We've passed `self` as a parameter to this method and then don't use it. If we just want to use teh `pytest.raises(xxx)` method, there's no need for the class usage and injecting `self`.

Reply:
i changed this function .should be fixed now.